lib/inquisitor.ex

```diff
@@ -78,10 +78,20 @@ defmodule Inquisitor do
       end
 
       defp unquote(fn_name)(query, []), do: query
+
       defp unquote(fn_name)(query, [{attr, value}|tail]) do
-        query
```

Comment:
`limit` shouldn't be handled in this function. It should be handled in its own function